src/Entity/Mooc/BaseMoocElement.php

```diff
@@ -47,24 +57,63 @@
     protected $slug;
 
     /**
-     * This attribute should be overridden in the child class for the SortablePosition.
+     * @var int
      *
-     * @see AppBundle\Entity\Mooc\Video::$displayOrder
-     * @see https://github.com/Atlantic18/DoctrineExtensions/blob/master/doc/sortable.md
+     * @ORM\Column(type="smallint")
+     * @Gedmo\SortablePosition
      */
-    protected $displayOrder;
+    protected $position;
```

Comment:
Why this renaming ?

Reply:
because it's a position, it's clearer than displayOrder, and it's the common naming field. By default bundles like pixsortablebundle use field name position.